app/views/va_facilities/_facility_map.html.erb

```diff
@@ -0,0 +1,29 @@
+<script>
```

Comment:
if this html file is just running javascript to load the map can we implement this map similar to the `visns_index_map`? 🤔